content/en/serverless/distributed_tracing/collect_lambda_payloads.md

```diff
@@ -71,6 +71,8 @@ DD_APM_REPLACE_TAGS=[
 ]
 ```
 
+Please note, payload obfuscation is supported only via the Datadog Lambda Extension, and is not available when used in conjunction with the Datadog Forwarder.
```

Comment:
Payload obfuscation is available in when using the Datadog Lambda Extension. It is not available in the Datadog Forwarder.